src/Parser/Rogue/Assassination/Modules/Features/Abilities.js

```diff
@@ -13,14 +13,14 @@ class Abilities extends CoreAbilities {
       {
         spell: SPELLS.VENDETTA, // TODO: Reduced by Convergence of Fates
         category: Abilities.SPELL_CATEGORIES.COOLDOWNS,
-        cooldown: 180,
+        cooldown: (_, combatant) => 120 - combatant.owner.modules.masterAssassin.traitCooldownReduction,
```

Comment:
it's preferred not to get at a module through `owner.modules`. Instead, at the top of the class add
```
static dependencies = {
  masterAssassin: MasterAssassin;    
};
```
and then here you can get the same value with `this.masterAssassin.traitCooldownReduction`

Reply:
This looks an awful lot like my trueshot reduction code, so probably my fault 😅

Reply:
( ఠ ͟ʖ ఠ)